README.md

```diff
@@ -12,15 +12,24 @@ The SharePoint Backend allows administrators to add SharePoint document librarie
 
 ![screenshot](screenshots/configuration.png)
 
-Supports SharePoint 2013, 2016, 2019 (experimental) and SharePoint Online (Office 365). Nextcloud accesses SharePoint through the SharePoint REST API and uses SAML Token authentication, with a fallback to NTLM auth. Nextcloud respects file access permissions associated with its configured user credentials. Versioning and sharing are handled by Nextcloud.
+* Supports SharePoint 2013, 2016, 2019 (experimental) and SharePoint Online (Office 365).
+* Nextcloud accesses SharePoint through the SharePoint REST API and uses SAML Token authentication, with a fallback to NTLM auth.
+* Nextcloud respects file access permissions associated with its configured user credentials.
+* Versioning and sharing are handled by Nextcloud.
 
-# How To Add Office 365 Onedrive
-  Make sure you have enabled [External storage support](https://docs.nextcloud.com/server/24/go.php?to=admin-external-storage) and the Sharepoint app. Then go to Administration 
-  Settings and select External Storage and add Sharepoint as follow:
-  First, you need to get your remote's URL:
+Learn more about External Storage and SharePoint on [https://nextcloud.com/storage/](https://nextcloud.com/storage/).
 
-Go [here](https://onedrive.live.com/about/en-us/signin/) to open your OneDrive or to sign in
-Now take a look at your address bar, the URL should look like this: https://[YOUR-DOMAIN]-my.sharepoint.com/personal/[YOUR-EMAIL]/_layouts/15/onedrive.aspx
-You'll only need this URL up to the email address as "Host". After that, you'll most likely want to add "Documents" as "Document Library". That subdirectory contains the actual data stored on your OneDrive. Then enter your email as "Username" and password and you are good to go. Credit goes to [rclone](https://rclone.org/webdav/#sharepoint-online).
+# How To Add Office 365 Onedrive
 
-Learn more about External Storage and SharePoint on [https://nextcloud.com/storage/](https://nextcloud.com/storage/)
+1. Enable the *[External storage support](https://docs.nextcloud.com/server/stable/go.php?to=admin-external-storage)* app.
+2. Install/enable the *Sharepoint Backend* app (this app).
+3. Get your remote's URL:
+    1. Go [here](https://onedrive.live.com/about/en-us/signin/) to open your OneDrive or to sign in.
+    2. Look at your address bar. The URL should look like this: `https://[YOUR-DOMAIN]-my.sharepoint.com/personal/[YOUR-EMAIL]/_layouts/15/onedrive.aspx`. You'll only need this URL up to the email address.
+    3. You'll most likely need to append `Documents` to the URL. That subdirectory contains the actual data stored on your OneDrive.
```

Comment:
`Documents` should be entered in the `Document Library` field, not as part of the Host URL.